src/sql/azdata.proposed.d.ts

```diff
@@ -1175,6 +1175,10 @@ declare module 'azdata' {
 			 * The group name, properties with the same group name will be displayed under the same group on the UI.
 			 */
 			group?: string;
+			/**
+			 * Whether the property should be displayed in the properties view. The default value is true.
+			 */
+			showInPropertiesView?: boolean;
```

Comment:
we will use this to:
1. mark the top levels tables to not show up in properties view
2. hide some properties in properties view, e.g. the data type display column in the columns table which consists information of  datatype, length, precision, scale. same as in SSDT/SSMS.